zib/StructureDefinitions/zib-BarthelADLIndex.xml

```diff
@@ -0,0 +1,412 @@
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="273302005" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueInteger">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueInteger" />
+      <short value="TotalScore" />
+      <definition value="The Barthel ADL index is the sum of the scores of the individual variables. The interpretation of this total score is 0-9 for serious dependence, 10-19 for slight dependence and 20 for independence. Twenty is the maximum score." />
+      <comment value="The individual scores are communicated using the different `Observation.component` slices. The ordinal values of these indivdual scores, which are used to calculate the total score, are defined by the underlying code system and are registered in the ValueSet resources bound on these slices, using the [ordinalValue](http://hl7.org/fhir/StructureDefinition/ordinalValue) extension." />
+      <alias value="TotaalScore" />
+      <type>
+        <code value="integer" />
+      </type>
+      <minValueInteger value="0" />
+      <maxValueInteger value="20" />
+      <mapping>
+        <identity value="zib-bartheladlindex-v3.2-2024EN" />
+        <map value="NL-CM:4.2.2" />
+        <comment value="TotalScore" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:bowels">
+      <path value="Observation.component" />
+      <sliceName value="bowels" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:bowels.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1" />
+          <code value="4002004" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:bowels.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="Bowels" />
+      <definition value="The bowel function describes the extent of fecal incontinence. The measurement takes the condition as measured in the preceding week. In this scale, the following interpretations are used:" />
```

Comment:
This definition is now missing a part of the definition from the zib. This is also true for the other components. Do you have any idea why?

Reply:
Fixed